loaders/src/glTF/2.0/Extensions/KHR_lights_punctual.ts

```diff
@@ -114,6 +130,8 @@ export class KHR_lights implements IGLTFLoaderExtension {
 
                 GLTFLoader.AddPointerMetadata(babylonLight, extensionContext);
 
+                this._assignExtras(babylonLight, node);
```

Comment:
I don't understand this change. The parent node will already have all the extras on it via the ExtrasAsMetadata loader extension. This seems like duplicate information?

Reply:
Indeed. Accessing the extras/metadata on the light itself looks more intuitive to me but you certainly had such discussion before. Tell me if you want me to remove the extras from the light.

Reply:
I think for consistency sake, it should just be on the parent node so that it matches the glTF.

Reply:
Ok!

Reply:
If the light extension itself had extras, then it might make sense to do what you are doing here but for the light extension instead of the node.

Reply:
Done in https://github.com/BabylonJS/Babylon.js/pull/7252/commits/7c892e0aae5ae7606fa23989738d49c61e182b6b